Sources/ConsoleKitAsyncExample/entrypoint.swift

```diff
@@ -16,7 +16,6 @@ struct AsyncExample {
             try await console.run(group, input: input)
         } catch let error {
             console.error("\(error)")
-            exit(1)
```

Comment:
Why not `exit(1)` anymore?

Reply:
'cause it had no meaningful effect, and exiting with a nonzero status is arguably a questionable behavior to provide as an example (especially when `exit(1)` isn't a fully platform-agnostic or even necessarily correct means of doing so)